beacon_node/eth2-libp2p/src/rpc/protocol.rs

```diff
@@ -23,7 +26,33 @@ use tokio_util::{
     codec::Framed,
     compat::{Compat, FuturesAsyncReadCompatExt},
 };
-use types::EthSpec;
+use types::{BeaconBlock, EthSpec, Hash256, MainnetEthSpec, Signature, SignedBeaconBlock};
+
+lazy_static! {
+    pub static ref SIGNED_BEACON_BLOCK_MIN: usize = SignedBeaconBlock::<MainnetEthSpec> {
```

Comment:
@AgeManning is it okay here that I'm sorta hardcoding `MainnetEthSpec`? I couldn't figure out how to keep it generic over `EthSpec`.

Reply:
This looks fine to me. 
Add a comment indicating we are doing this and that the MAX/MIN values are the same across each spec type